specification/oracle/Oracle.Database/operations/oraclesubscription/oracleSubscriptionOperations.tsp

```diff
@@ -0,0 +1,80 @@
+import "../../../models/oraclesubscription/oracleSubscription.tsp";
+
+using Azure.ResourceManager;
+using Azure.ResourceManager.Foundations;
+using Autorest;
+
+namespace Oracle.Database;
+
+@armResourceOperations(OracleSubscription<Plan>)
+interface OracleSubscriptions
+  extends OracleSubscriptionLevelResource<
+      OracleSubscription<Plan>,
+      OracleSubscriptionProperties
+    > {
+  @doc("List Cloud Account Details")
+  @example(
+    "./examples/oracleSubscriptions_listCloudAccountDetails.json",
+    "List Cloud Account details for the Oracle Subscription"
+  )
+  listCloudAccountDetails is ListActions.ArmResourceGetActionAsync<
+    OracleSubscription<Plan>,
+    CloudAccountDetails
+  >;
+
+  @doc("List Saas Subscription Details")
+  @example(
+    "./examples/oracleSubscriptions_listSaasSubscriptionDetails.json",
+    "List Saas Subscription details for the Oracle Subscription"
+  )
+  listSaasSubscriptionDetails is ListActions.ArmResourceGetActionAsync<
```

Comment:
list sounds like a paging operation not an lro.

Reply:
That is a good catch!
Initially it was supposed to be used as asynchronous operation. We will take a look on that and reconsider if it should be synchronous paging operation on our side.
Since this is a preview version and it actually works can we get it fixed with the next PR to not block our other development threads?